pkg/kubelet/container/helpers.go

```diff
@@ -91,7 +91,7 @@ func ShouldContainerBeRestarted(container *v1.Container, pod *v1.Pod, podStatus
 // HashContainer returns the hash of the container. It is used to compare
 // the running container with its desired spec.
 func HashContainer(container *v1.Container) uint64 {
-	hash := adler32.New()
+	hash := fnv.New32a()
```

Comment:
@resouer can you leave out the changes to files in `pkg/kubelet`. We'll handle it in a separate PR. Thanks!